src/colinajose/service/Observer.java

```diff
@@ -0,0 +1,5 @@
+package colinajose.service;
+
+public interface Observer {
+    void update();
```

Comment:
Probably this can have some sort of message to be sent.

Reply:
@JurgenPadilla the whole implementation of this will be in a future commit